src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageStrings.cs

```diff
@@ -82,15 +82,14 @@ public static string Option_Show_hints_for_implicit_object_creation
         public static string Option_DisplayLineSeparators
             => CSharpVSResources.Show_procedure_line_separators;
 
+        public static string Option_Underline_reassigned_variables
+            => ServicesVSResources.Underline_reassigned_variables;
```

Comment:
You might want to add the text in this file to make it searchable in options page
https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/CSharp/Impl/VSPackage.resx

Reply:
> You might want to add the text in this file to make it searchable in options page
https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/CSharp/Impl/VSPackage.resx

Will do.